src/reducers/combined-updaters.js

```diff
@@ -78,10 +81,47 @@ const combinedUpdaters = null;
 
 export const isValidConfig = config =>
   isPlainObject(config) && isPlainObject(config.config) && config.version;
+
 export const defaultAddDataToMapOptions = {
   centerMap: true,
   keepExistingConfig: false
 };
+
+const identity = state => state;
+
+/* eslint-disable-next-line no-unused-vars */
+function log(text) {
+  return value => Console.log(text, value);
+}
+
+function payload_(p) {
```

Comment:
why the "_" at the end?

Reply:
to make them consistent, because some of these functions have similar names as lodash, if we import pick from 'lodash.pick' we would have conflict with our own pick_ function